src/07_slices.py

```diff
@@ -2,36 +2,34 @@
 Python exposes a terse and intuitive syntax for performing 
 slicing on lists and strings. This makes it easy to reference
 only a portion of a list or string. 
-
 This Stack Overflow answer provides a brief but thorough
 overview: https://stackoverflow.com/a/509295
-
 Use Python's slice syntax to achieve the following:
 """
 
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2][0])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a) - 2])
```

Comment:
Totally understand doing it this way coming from JS; Python does give you the option of negative indexing, so you could tighten it up to `print(a[-2])`